internal/app/connectconformance/server_runner.go

```diff
@@ -176,7 +176,7 @@ func runTestCasesForServer(
 				results.assert(name, expectations[resp.TestName], resp.GetResponse())
 			}
 			if isReferenceClient {
-				for _, msg := range resp.Feedback {
+				for _, msg := range resp.GetFeedback() {
```

Comment:
This is an unrelated fix. I encountered a nil-pointer-defereference here at one point while developing this arc of work in #682. Since this PR has other changes in the test runner, seemed okay to drop this here.